.github/workflows/pr-update-playwright-expectations.yaml

```diff
@@ -60,25 +60,22 @@ jobs:
         uses: actions/checkout@v5
         with:
           ref: ${{ steps.pr-info.outputs.branch }}
-
-      # Setup Test Environment, build frontend but do not start server yet
-      - name: Setup ComfyUI server
-        uses: ./.github/actions/setup-comfyui-server
       - name: Setup frontend
         uses: ./.github/actions/setup-frontend
         with:
           include_build_step: true
-      - name: Setup Playwright
-        uses: ./.github/actions/setup-playwright
-
-      # Save the entire workspace as cache for update jobs to restore
+      # Save expensive build artifacts (Python env, built frontend, node_modules)
+      # Source code will be checked out fresh in sharded jobs
       - name: Generate cache key
         id: cache-key
         run: echo "key=$(date +%s)" >> $GITHUB_OUTPUT
       - name: Save cache
         uses: actions/cache/save@5a3ec84eff668545956fd18022155c47e93e2684
         with:
-          path: .
+          path: |
+            ComfyUI
+            dist
+            node_modules
```

Comment:
If you're using the built files, do you need node_modules?
And ComfyUI is just the checked out Comfy Repo, which might be faster to checkout than to restore from cache.